website/docs/components/icon.mdx

```diff
@@ -33,35 +34,66 @@ import { Icon } from "@chakra-ui/core"
 
 ## Usage
 
-Use an icon by passing the `name` prop. This name must match an icon key in
-`theme.icons`. By default, the icon inherits the `fontSize` and `color` of its
-parent.
+Use an icon by directly importing it with its respective name as mentioned in
+[next section](#all-icons). By default, Icon component inherits `fontSize` and
+`color` of its parent.
 
 ```jsx
-<Stack direction="row">
+<HStack>
   {/* Default size is 1em => 16px */}
   <PhoneIcon />
 
-  {/* Use the `size` prop to change the icon size */}
-  <CheckCircleIcon size="24px" />
```

Comment:
Hope using size is not recommended and confusing. So, used boxSize in the examples.